storage/mysql/admin_storage.go

```diff
@@ -295,7 +295,7 @@ func (t *adminTX) CreateTree(ctx context.Context, tree *trillian.Tree) (*trillia
 		newTree.TreeType.String(),
 		"RFC6962_SHA256",
 		newTree.HashAlgorithm.String(),
-		newTree.SignatureAlgorithm.String(),
+		"ECDSA",
```

Comment:
Do you need to do something about L48 and L272?

Additionally, how do current users deal with the schema change?

Reply:
This is why put the string "ECDSA" here, to avoid a schema change, but still putting in something that's valid (previous PRs have been narrowing down the selection of signature algorithms to just this one, so that this is likely to be effectively what would have been written before).

In the query, I could drop it, but I didn't want to mess about with this too much...

Reply:
Perhaps leave a TODO() or at least a note on the field references, noting it's deprecated and unused for future readers. Thanks.

Reply:
Done.